shanoir-ng-datasets/src/main/java/org/shanoir/ng/importer/service/ImporterService.java

```diff
@@ -380,6 +380,7 @@ private boolean checkSerieForDicomImages(Serie serie) {
     }
 
     public void cleanTempFiles(String workFolder) {
```

Comment:
Maybe this method should be remove altogether ?

Reply:
No, but it should absolutely not be commented ! thanks !